gapic/utils/lines.py

```diff
@@ -40,6 +40,47 @@ def sort_lines(text: str, dedupe: bool = True) -> str:
     return f'{leading}{answer}{trailing}'
 
 
+def get_subsequent_line_indentation_level(list_item: str) -> int:
+    """
+    Given a list item return the indentation level for subsequent lines.
+    For example, if it is a numbered list, the indentation level should be 3
+    as shown below.
+
+    Here subsequent lines should be indented by 2
+
+    - The quick brown fox jumps over the lazy dog. The quick brown fox jumps
+      over the lazy dog
+
+    Here subsequent lines should be indented by 2
+
+    + The quick brown fox jumps over the lazy dog. The quick brown fox jumps
+      over the lazy dog
+
+    Here subsequent lines should be indented by 3
+
+    1. The quick brown fox jumps over the lazy dog. The quick brown fox jumps
+       over the lazy dog
+
+    """
+    if len(list_item) >= 2 and list_item[0:2] in ['- ', '+ ']:
+        indentation_level = 2
+    elif len(list_item) >= 3 and list_item[0].isdigit() and list_item[1:3] == '. ':
```

Comment:
This isn't robust if we get into double digits. How about searching that anything preceding the first space is a series of digits followed by a period?
```suggestion
    elif re.match("^\d+\. ", list_item):
```

Haven't tested this, and we may want to pre-define the pattern for efficiency.

Reply:
Based on local testing, the sphinx docs build will fail with double digits. This could be a bug in sphinx. I don't think it's worth while supporting double digits if sphinx can't handle it. We don't have any APIs using numbered lists with double digits so I'm not sure if it's worthwhile to go down the rabbit hole.

Reply:
> Based on local testing, the sphinx docs build will fail with double digits.

I'm surprised. The [Sphinx spec](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks) does not mention double-digit items but implies they are supported. Is it a problem with our testing?

I think we should still implement my suggestions about regexes, because then at least _this_ code you're changing will be future-proof and work when we encounter a double-digit list. This way we don't accrue more technical debt. All that remains will be figuring out why the docs tests fail with double-digit items, which we can track and fix separately (and lower priority for now since we haven't encountered it in the field, as you say).